src/main/webapp/dev/js/common/feedbackResponseComments.es6

```diff
@@ -376,6 +393,12 @@ function hideResponseCommentAddForm(recipientIndex, giverIndex, qnIndex, section
         $(`#responseCommentTable${id}`).css('margin-top', '0');
         $(`#responseCommentTable${id}`).hide();
     }
+    const addFormTable = $(`#showResponseCommentAddForm${id} > form`).find('table').first();
+    const checkboxesInAddFormTable = addFormTable.find('tr').find('input.visibilityCheckbox');
+    const valueOfCheckboxes = getInitialVisibilityOfCheckboxes($(`#showResponseCommentAddForm${id}`));
```

Comment:
Perhaps you can extract all these checks into another function? They are being done more than once.